core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -64,8 +65,16 @@ package object config {
       .stringConf
       .createOptional
 
+  private[spark] val DRIVER_DEFAULT_EXTRA_CLASS_PATH =
+    ConfigBuilder(SparkLauncher.DRIVER_DEFAULT_EXTRA_CLASS_PATH)
+      .internal()
+      .version("4.0.0")
+      .stringConf
+      .createWithDefault(SparkLauncher.DRIVER_DEFAULT_EXTRA_CLASS_PATH_VALUE)
+
   private[spark] val DRIVER_CLASS_PATH =
     ConfigBuilder(SparkLauncher.DRIVER_EXTRA_CLASSPATH)
+      .withPrepended(DRIVER_DEFAULT_EXTRA_CLASS_PATH.key, File.pathSeparator)
```

Comment:
Hmm, so looks like users can simply set `SparkLauncher.DRIVER_EXTRA_CLASSPATH` to achieve same effect, no?

Reply:
Yes, right. However, it's a breaking change if we enforce to use it.
>  so looks like users can simply set SparkLauncher.DRIVER_EXTRA_CLASSPATH to achieve same effect, no?

Reply:
Given that a company has many many Spark job, it's not simple because we need to add `SparkLauncher.DRIVER_EXTRA_CLASSPATH` and `SparkLauncher.EXECUTOR_EXTRA_CLASSPATH` for every Spark jobs.

Reply:
Got it. Thanks.